### PR TITLE
Refactor/build settings

### PR DESCRIPTION
**Pull request by cservan**
Refactored version in which the build and installation and the API work through Docker by using the install.sh script.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,26 +1,46 @@
...
 cmake_minimum_required(VERSION 3.5)
 
 set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -lpthread")
 set(CMAKE_VERBOSE_MAKEFILE ON)
 set(CMAKE_BUILD_TYPE RelWithDebInfo)
 
-include_directories(${PROJECT_SOURCE_DIR}/include/ /usr/local/include/)
-link_directories(/usr/local/lib/)
+include_directories(${PROJECT_SOURCE_DIR}/include/ /usr/local/include/ /usr/)
+#include_directories(${PROJECT_SOURCE_DIR}/vendor/qnlp-toolkit/vendor/fastText/)
```
Comment (maudetes) [inline CMakeLists.txt]: Remove commented lines

File: Dockerfile [hunk 2]
```diff
@@ -2,7 +2,9 @@
 
 FROM ubuntu:18.04
 
-LABEL maintainer="n.martin@qwantresearch.com"
+LABEL maintainer="c.servan@qwantresearch.com"
```
Comment (maudetes) [inline Dockerfile]: Change to e.maudet instead

**Review by maudetes — COMMENTED** [review]
Seems fine, few things need to be changed.

File: CMakeLists.txt (hunk 1 above)
Comment (cservan) [reply CMakeLists.txt]: Commented lines removed

File: Dockerfile (hunk 2 above)
Comment (cservan) [reply Dockerfile]: removed maintaner to add authors instead.

File: install.sh [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+export CMAKE_PREFIX_PATH=$PREFIX
+
+git submodule update --init --recursive
+
+echo "Installing dependencies"
+
+pushd vendor/qnlp-toolkit
+rm -rf build
+git pull  --recurse-submodules 
+bash install.sh
```
Comment (maudetes) [inline install.sh]: missing prefix

Comment (cservan) [reply install.sh]: Done

**Pull request merged**